src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -739,59 +730,29 @@ private void Logic_NodeMessagesClearing(NodeModel obj)
             // errors aren't being dismissed when the graph runs.
             if (nodeLogic.State == ElementState.Error) return;
 
-            ErrorBubble.NodeMessages.Clear();
+            if (DynamoViewModel.UIDispatcher != null)
+            {
+                DynamoViewModel.UIDispatcher.Invoke(() =>
+                {
+                    ErrorBubble.NodeMessages.Clear();
+                });
+            }
```

Comment:
I dont think UIDispatcher will always exist, we have found that when running unit tests, it will be null

Reply:
It was something we discussed with @mjkkirschner, we get a runtime exception (Dynamo instantly crashes) when trying to affect an ObservableCollection from the main thread.

Reply:
I understand, just saying when the UIDispatcher is null, we may need to fall back. Not sure if you remember I had to add that for you to address some regressions previously. We will get back to this

Reply:
I've patched this up as per the previous one.